llvm/lib/DebugInfo/Symbolize/DIPrinter.cpp

```diff
@@ -131,14 +131,17 @@ void PlainPrinterBase::printFunctionName(StringRef FunctionName, bool Inlined) {
 
 void LLVMPrinter::printSimpleLocation(StringRef Filename,
                                       const DILineInfo &Info) {
-  OS << Filename << ':' << Info.Line << ':' << Info.Column << '\n';
+  OS << Filename << ':' << Info.Line << ':' << Info.Column
+     << (Info.IsApproximatedLine ? (" " + Twine(Info.ApproxString)) : "")
```

Comment:
Perhaps this (& below) would be more legible following the way `Discriminator` is printed, using an `if` rather than a conditional operator:
```
OS << Filename << ':' << Info.Line << ':' << Info.Column
if (Info.IsApproximatedLine)
  OS << " " << Info.ApproxString;
OS << '\n';
```
(& same change below in `GNUPrinter::printSimpleLocation`)

Reply:
Done